extensions/tabs/scripts/confusion.js

```diff
@@ -0,0 +1,340 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// d3 and json2html included in HTML script reference
+/* global d3, json2html */
+
+// json2html uses curly brace syntax for templating
+/* eslint no-template-curly-in-string: "warn" */
+
+const margin = { top: 20, right: 20, bottom: 20, left: 20 }
+const padding = { top: 60, right: 60, bottom: 60, left: 60 }
+const outerWidth = 960
+const outerHeight = 500
+const innerWidth = outerWidth - margin.left - margin.right
+const innerHeight = outerHeight - margin.top - margin.bottom
+const width = innerWidth - padding.left - padding.right
+const height = innerHeight - padding.top - padding.bottom
+var nestData, sliceActualIntent, sliceExpectedIntent, xScale, yScale
```

Comment:
Still curious if there's a way to encapsulate these so they don't get set as globals on the `window` object. Presumably these variables should be scoped to the `getData` function.